src/ai/backend/manager/clients/storage_proxy/session_manager.py

```diff
@@ -177,7 +178,11 @@ async def _fetch(
             proxy_name: str,
             client: StorageProxyManagerFacingClient,
         ) -> Iterable[tuple[str, VolumeInfo]]:
-            reply = await client.get_volumes()
+            try:
+                reply = await client.get_volumes()
+            except StorageProxyConnectionError:
+                log.warning("Failed to connect to storage proxy (name: {})", proxy_name)
+                return []
```

Comment:
It doesn't seem like this is the way to consume the error, is there a reason you worked this way?